python/nav/web/webfront/urls.py

```diff
@@ -33,6 +33,11 @@
     re_path(r'^index/logout/', views.logout, name='webfront-logout'),
     # Dashboard
     re_path(r'^index/dashboard/(?P<did>\d+)', views.index, name='dashboard-index-id'),
+    re_path(
+        r'^index/dashboard/toggle-shared/(?P<did>\d+)',
```

Comment:
This could be 
```suggestion
    path(
        'index/dashboard/toggle-shared/<int:did>',
```

You'll probably have to import path (same module as re_path) though.

Reply:
Rebasing...